build.gradle.kts

```diff
@@ -0,0 +1,43 @@
+import org.gradle.api.JavaVersion.VERSION_11
+
+version = "0.0.1-SNAPSHOT"
+group = "com.bisnode.opa"
+
+plugins {
+    groovy
+    java
+    `java-library`
+    `maven-publish`
+}
+
+repositories {
+    jcenter()
```

Comment:
What is it we retrieve from here?

Reply:
Actually there is no sense to use jcenter here, we can use mavenCentral